python/run_inline_query.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import looker_sdk
+
+
+sdk = looker_sdk.init31()
+
+fields = [
+    'hits_eventInfo.eventAction',
+    'hits_eventInfo.eventCategory',
+]
```

Comment:
Perhaps useful to clarify `fields` and `sort_order` take input as `<model>.<dimension|measure>`.